ci-operator/config/stackrox/collector/stackrox-collector-master.yaml

```
@@ -11,10 +20,50 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: lint
-  commands: "true"
-  container:
-    from: src
+- as: lint-via-clang-format
+  steps:
+    test:
+    - as: job
+      commands: make -C collector check
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
+- as: lint-via-shfmt
+  steps:
+    test:
+    - as: job
+      commands: go install mvdan.cc/sh/v3/cmd/shfmt@v3.4.1; make shfmt-check
```

shfmt is in collector-test. 
```suggestion
      commands: make shfmt-check
```